src/kani-compiler/rustc_codegen_kani/src/utils/names.rs

```diff
@@ -109,6 +109,10 @@ impl<'tcx> GotocCtx<'tcx> {
         // TODO: use def_id https://github.com/model-checking/kani/issues/364
         idx.to_string().into()
     }
+
+    pub fn reach_check_msg(check_id: &str) -> String {
```

Comment:
Also, this function name is confusing.

Reply:
The documentation I added should hopefully clarify what this function is doing. I'm not good with picking names, so I'm open for suggestions.

Reply:
reachability_check_message?

Reply:
Updated.